### ISSUE TITLE
Offices not spawning

### ISSUE THREAD

**Issue by Asher5290** [johnytoxic/CSL2DemandControl#9]
So I have run into the issue where the office demand bar is full, but there are no factors affecting demand like in the other demand bars and offices do not spawn at all. Even though the road is green and the demand bar is full. I checked for errors in the Bepinix log but couldn't find any. Do you know how to fix this by chance? Your mod is the only reason I still play this game.

**Comment by johnytoxic** [comment]
Yes this is a known issue. This also affects other sectors (see #2). There are lots of tiny screws which need to be adjusted.

Unfortunately, I couldn't find the time to investigate and test that, so far.

**Comment by pixelinkmedia** [comment]
This happens to me too.
However, I was able to do some offices.
But, now no more will build.
Thanks
